lib/providers/filesystem/index.js

```diff
@@ -3,6 +3,10 @@
 // This file is licensed under the Artistic License 2.0.
 // License text available at https://opensource.org/licenses/Artistic-2.0
 
+// Turning on strict for this file breaks;
+// disabling strict for this file
+/* eslint-disable strict */
+
```

Comment:
We'll leave the fix out of this PR as discussed. Please submit another PR and enable use strict + parseInt('0666', 8) for the octal value that causes npm test to fail before running the linter.